autogen.sh

```diff
@@ -5,6 +5,9 @@
 : ${srcdir:=.}
 : ${SAVE_DIST_FILES:=0}
 
+if [ -z "$MAKE" ]; then
```

Comment:
Why don't you use `: ${MAKE:=make}` ?

Reply:
> Why don't you use `: ${MAKE:=make}` ?

Just I'm not familiar with make.

Reply:
I ask to use `: ${MAKE:=make}` instead of `if [ -z "$MAKE" ] ; then`.